src/components/notificationsCenter/conversationPreview.styles.scss

```
@@ -0,0 +1,80 @@
+		color: var(--m3-on-surface-variant, #444748);
+	}
+
+	&__bubble {
+		background: var(--m3-surface-container-high, #eae7e8);
+		color: var(--m3-on-surface, #1b1b1c);
+		border-radius: 12px;
+		padding: 8px 12px;
+		font-size: 14px;
+		line-height: 1.5;
+		white-space: pre-wrap;
+		word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)